component/prometheus/relabel/relabel.go

```diff
@@ -46,15 +46,15 @@ type Arguments struct {
 	MetricRelabelConfigs []*flow_relabel.Config `river:"rule,block,optional"`
 
 	// Cache size to use for LRU cache.
-	//CacheSize int `river:"cache_size,attr,optional"`
+	CacheSize int `river:"max_cache_size,attr,optional"`
```

Comment:
We probably want to validate this is not negative. I wonder if 0 has a meaning of unlimited and if we should allow it at all? Seems like better not to allow it, but I could be convinced otherwise.

Reply:
0 and below basically means the lru cache always evicts after adding.

Reply:
will add a check for 0